lucene/build.gradle

```diff
@@ -15,8 +15,56 @@
  * limitations under the License.
  */
 
+// Should we do this as :lucene:packaging similar to how Solr does it?
```

Comment:
Yes, please move it to :lucene:packaging. It makes things much cleaner in the long run.